src/Apps/IndustrialMetaverse/Site/Site.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { ISiteProps } from './Site.types';
+
+const Site: React.FC<ISiteProps> = ({ extensionClient }) => {
+    if (extensionClient) {
+        return <div>Site</div>;
+    } else {
+        return <div>Site without client</div>;
```

Comment:
great idea.

Reply:
We should be able to leverage react-router-dom hook `useLocation` to get any pathing and query string parameters to pass from here into the various sub-explorers